snippets/sdk/python/configurations/archive_file.py

```diff
@@ -0,0 +1,61 @@
+#! /usr/local/env python3
+
+import base64
+import os
+
+from azure.identity import DefaultAzureCredential
+from azure.mgmt.nginx import NginxManagementClient
+
+
+def main():
+    SUBSCRIPTION_ID = os.environ.get("AZURE_SUBSCRIPTION_ID", None)
+    GROUP_NAME = "myResourceGroup"
+    DEPLOYMENT_NAME = "myDeployment"
+    CONFIGURATION_NAME = "default"  # configuration must be named default
```

Comment:
does this requirement need to be documented anywhere else?

Reply:
Should be documented in both snippets examples. I really wish we did not require the configuration name and it just defaulted to....default.